CHANGELOG.md

```diff
@@ -88,6 +88,7 @@ And please only add new entries to the top of this list, right below the `# Unre
   `DeviceEvent::Key` support.
 - **Breaking** `MouseButton` now supports `Back` and `Forward` variants, emitted from mouse events
   on Wayland, X11, Windows, macOS and Web.
+- On Web, do not destroy the event loop on `pagehide` events.
```

Comment:
No need to add a new changelog entry, we didn't have a release since the last change to this. We should fold any more details into the last changelog entry about this.
https://github.com/rust-windowing/winit/blob/f9f42fbc4753933158c2090a2948f9aa1776ca37/CHANGELOG.md?plain=1#L79

Reply:
Thanks, updated this!